### PR TITLE
refactor(core/error): use evaluated call sites for formatting

### PR DESCRIPTION
**Pull request by nayeemrmn**
small cleanup

### CODE REVIEW

File: core/01_core.js [hunk 1]
```diff
@@ -17,6 +17,7 @@
     Promise,
     ObjectFromEntries,
     MapPrototypeGet,
+    MapPrototypeHas,
```
Comment (bartlomieju) [inline core/01_core.js:20]: Why this change?

Comment (nayeemrmn) [reply core/01_core.js:20]: It's a missing import used at https://github.com/denoland/deno/blob/d81c5b51b32247c96af78d90e89429e33b2a58ac/core/01_core.js#L102 Forgot that change was in my workspace for this, oh well.

**Review by bartlomieju — APPROVED** [review]
LGTM, good cleanup
